cmd/meroxa/turbine/ruby/build.go

```diff
@@ -0,0 +1,16 @@
+package turbinerb
+
+import (
+	"context"
+)
+
+func (t *turbineRbCLI) Build(ctx context.Context, appName string, platform bool) (string, error) {
+	return "", nil
```

Comment:
I think these should return a "not implemented" error

Reply:
Sounds good!